backend/tests/routes/teams/external_invitations.rs

```diff
@@ -0,0 +1,77 @@
+#![cfg(feature = "full-test-suite")]
+use crate::test_utils::TestApp;
+use crate::test_utils::database::TestDatabase;
+use crate::test_utils::header::TestAuthHeader;
+use crate::test_utils::mail::SmtpTestClient;
+use actix_web::test;
+use hack4krak_backend::entities::sea_orm_active_enums::UserRoles;
+use hack4krak_backend::entities::users;
+use serde_json::json;
+
+#[actix_web::test]
+async fn external_invitations_flow() {
+    let test_database = TestDatabase::new().await;
+    let smtp_client = SmtpTestClient::new().await;
+    let user = test_database.with_default_user().await;
+    let admin = test_database
+        .with_user(users::UpdatableModel {
+            roles: Some(UserRoles::Admin),
+            ..Default::default()
+        })
+        .await;
+
+    let app = TestApp::default()
+        .with_database(test_database)
+        .with_smtp_client(smtp_client.smtp_client.clone())
+        .build_app()
+        .await;
+
+    // Register organization as admin
+    let request = test::TestRequest::post()
+        .uri("/admin/email/send_external_registration_form")
+        .set_json(json!({
+            "organization": "School for Silly Crabs".to_string(),
+            "email_address": "crab@norbiros.dev".to_string(),
+        }))
+        .insert_header(TestAuthHeader::new(admin.clone()))
+        .to_request();
+    let response = test::call_service(&app, request).await;
+    assert_eq!(response.status(), 200);
+
+    let confirmation_code = smtp_client.find_uuid_in_first_email().await;
+
+    let request = test::TestRequest::post()
+        .uri(&format!(
+            "/teams/external_invitations/create/{}",
+            confirmation_code
+        ))
+        .set_json(json!({
+            "teams": [["Kotki", 3], ["Kraby", 1]]
```

Comment:
```suggestion
            "teams": [["Kocieta", 3], ["Szczenieta", 1]]
```

As we well know from the guys in Pidżama Porno. In this world, there are only kittens and puppies.